release notes/v0.26.0.md

```diff
@@ -0,0 +1,120 @@
+  - correctly handle server initiated close (#1186)
+* JSON: Better error messages when parsing JSON fails. Now telling you at which line and row the
+  error is instead of just the offset. Thanks, @openmohan (#905)
+* HTTP: Use Request's `GetBody` in order to be able to get the body multiple times for a single
+  request as needed in 308 redirects of posts and if the server sends GOAWAY with no error (#1093)
+* JS: Don't export internal go struct fields of script options (#1151)
+* JS: Ignore `minIterationDuration` for `setup` and `teardown` (#1175)
+* UX: Better timeout messages for `setup` and `teardown` timeouts including a hint on how to fix it (#1173)
+* HTTP: Return error on any request that returns 101 status code as k6 currently doesn't support any protocol upgrade behaviour (#1172)
+* HTTP: Correctly capture TCP reset by peer and broken pipe errors and give them the appropriate error_code (#1164)
+* UX: Don't interpret not `K6_` prefixed env variables as k6 configuration, most notable `DURATION` and `ITERATIONS` (#1215)
+* JS/html: When using `Selection.map` was not wrapping the nodes it was outputting which lead to wrongly using the internal `Goquery.Selection` instead of k6's `Selection`. Thanks to @MMartyn for reporting this! (#1198)
+* HTTP: Correctly set the cookie for the current URL not the one the current request is redirecting to. Thanks @dimatock! (#1201)
+* Cloud: Add token to make calls to the cloud API idempotent (#1208)
+* Cloud: Improve aggregation of http metrics for different URLs but having the same `name` tag (#1220)
+* Cloud: Fix a bug where you weren't able to run a script, outputting to cloud, if it was using the shortcut urls for github/cdnjs (#1237)
+* UX: Include the path to the directory when a directory is used with `open()` (#1238)
+
+
+## Internals
+
+* CI: Stop using external service for testing websockets (#1138) and remove the last use of the external `httpbin.org`(#1213)
+* CI: Move to go 1.13 and back and to 1.13 again after a bug was fixed in the stdlib (#1145, #1157, #1177)
+* CI: Fix a test on MacOS (#1142)
+* CI: Fixing flaky tests (#1149, #1223)
+* Drop an external dependency for getting user's `configdir` (#1162)
+* Use bitmask for checking whether system tags are enabled, adding some small speedup where this is required (#1148)
+* Update `envconfig` as it was very old and the newer versions had fixes and features we want (#1214)
+* Metrics: Emit iterations as part of `netext.NetTrail`, instead of as a standalone one. Also cutting down on amount of bytes we sent to the cloud output (#1203)
+* UX: K6 Version will now include the git commit it was build from (in most cases ...) (#1235)
+
+## Breaking changes
+
+* Not really a breaking change, but `k6 convert` is getting deprecated in favor of [har-to-k6](https://github.com/loadimpact/har-to-k6). As such not new enchantment or bug fixes are expected for the internal converter. (#1228)
```

Comment:
```suggestion
* Not really a breaking change, but `k6 convert` is getting deprecated in favor of the newer [har-to-k6](https://github.com/loadimpact/har-to-k6) external converter. As such, no new enchantment or bug fixes are expected for the internal converter, and it will probably be completely removed in a few versions. (#1228)
```